### ISSUE TITLE
Could not install package 'MvvmCross.Plugin.PictureChooser 5.4.2' into a "Profile78" project

### ISSUE THREAD

**Issue by neminush** [MvvmCross/MvvmCross#2369]
### Expected behavior :thinking:
I would like to install a part of the plugin into portable library

### Actual behavior :bug:
Could not install package 'MvvmCross.Plugin.PictureChooser 5.4.2'. You are trying to install this package into a project that targets '.NETPortable,Version=v4.5,Profile=Profile78', but the package does not contain any assembly references or content files that are compatible with that framework. For more information, contact the package author.

### Configuration :wrench:

**Version:** 5.4.2

**Platform:** 
- [x] :portable library: .net v4.5, Profile78

**Comment by Cheesebaron** [comment]
PictureChooser is a Profile259 PCL. So it can't be consumed in Profile78 since 78 contains WP8.

Do you need to support WP8?

**Comment by neminush** [comment]
Thanks for the reply.
I don't know yet whether will I need support WP8.

**Comment by neminush** [comment]
I have changed profile for the PCL project from Profile78 to Profile259. Then I have tried to install the 'MvvmCross.Plugin.PictureChooser 5.4.2' and got the following message:

"Could not install package 'MvvmCross.Plugin.PictureChooser 5.4.2'. You are trying to install this package into a project that targets '.NETPortable,Version=v4.5,Profile=Profile259', but the package does not contain any assembly references or content files that are compatible with that framework. For more information, contact the package author."
